### DISCUSSION TITLE
pollin and reicht 2560 pro links

### DISCUSSION THREAD

**Discussion by mer2329** [sanni/cartreader#646] [General Chat]
was just looking at those links and noticed they are listing the 16AU processor. 
not sure if its right or not as the photos are of the old robotDYN board.
im not going to order from them as im in the USA and aliexpress is cheaper.

**Comment by dan-and** [comment]
Is there a updated ordering source for the the 2560-pro boards which does not have the AU variant?

Looks like all sellers just use old photos, but the 16AU chips :-(

**Reply by mer2329** [reply]
right now best bet is search for 2560 pro on the store of your choice, and base your buying decision off the title and pictures. theres always returns and buyer insurance if things dont match or work. just look for the resistor numbers and robotDyn logo. 

https://www.aliexpress.us/item/3256802655249815.html i recently bought that cause its title and pic both match, but there's still no guarantee its not the AU variant as sellers can do silent model swaps.
